src/game/AI/ScriptDevAI/scripts/outland/tempest_keep/arcatraz/arcatraz.cpp

```diff
@@ -160,6 +160,21 @@ void instance_arcatraz::OnCreatureCreate(Creature* creature)
             break;
     }
 }
+void instance_arcatraz::OnCreatureRespawn(Creature* creature)
+{
+    switch (creature->GetEntry())
+    {
+        case NPC_PROTEAN_NIGHTMARE:
+        case NPC_PROTEAN_HORROR:
+        if (creature->HasStringId(STRING_ID_ENTRANCE_GROUP))
```

Comment:
missing indent, if should be one tab further

Reply:
done